### PR TITLE
fix(websocket): fix websocket and memo

### PR DESCRIPTION
**Pull request by bdebon**
# What does this PR do?

[> Link to the JIRA ticket](https://qovery.atlassian.net/browse/FRT-430)

- Fixing a services table and environment table not updating the running status value when the websocket update.
It was due to the condition in the React.memo
- Fixing also some performances issue with the cohabitation of pulling deployment status and the running status websockets
- Fixing a react 18 warning about createRoot

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [x] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.  
Click on the link below to follow its deployment and use it.  
:point_right: [[PR] staging - fix(websocket): fix websocket and memo - 2022-06-15T08:31:20Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/bea637c7-8800-44d7-a1ee-537069526e8b/applications)

File: libs/pages/services/ui/src/lib/components/general/general.tsx [hunk 1]
```diff
@@ -97,11 +97,9 @@ export const GeneralPage = React.memo(General, (prevProps, nextProps) => {
   // Stringify is necessary to avoid Redux selector behavior
   const isEqual =
```
Comment (RemiBonnet) [inline libs/pages/services/ui/src/lib/components/general/general.tsx:98]: Cleaner to have one thing like this?
```  
const isEqual =
    JSON.stringify(
      prevProps.environments.map((environment) => ({
        status: environment.status?.state,
        running_status: environment.running_status?.state,
      }))
    ) ===
    JSON.stringify(
      nextProps.environments.map((environment) => ({
        status: environment.status?.state,
        running_status: environment.running_status?.state,
      }))
    )```

File: libs/pages/environments/ui/src/lib/components/general/general.tsx [hunk 2]
```diff
@@ -94,11 +94,9 @@ export const GeneralPage = React.memo(General, (prevProps, nextProps) => {
   // Stringify is necessary to avoid Redux selector behavior
   const isEqual =
```
Comment (RemiBonnet) [inline libs/pages/environments/ui/src/lib/components/general/general.tsx:95]: Same

**Review by RemiBonnet — COMMENTED** [review]
Thanks @bdebon !!
I have just one recommandation about the condition on the memo

File: libs/pages/services/ui/src/lib/components/general/general.tsx (hunk 1 above)
Comment (bdebon) [reply libs/pages/services/ui/src/lib/components/general/general.tsx:98]: yes indeed

**Pull request merged**